src/App.tsx

```diff
@@ -1,16 +1,29 @@
+import React from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [moviesList, setMoviesList] = React.useState(moviesFromServer);
+
+  const handelAddMovie = (movie: Movie): void => {
+    setMoviesList(prevState => {
+      return [
+        ...prevState,
+        movie,
+      ];
+    });
+  };
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={moviesList} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie onAdd={(movie) => handelAddMovie(movie)} />
```

Comment:
No real need to wrapper here
```suggestion
        <NewMovie onAdd={handelAddMovie} />
```